example/src/pages/LineRTCPage.tsx

```diff
@@ -103,6 +110,8 @@ const Line: React.FC<{}> = () => {
           </pre>
         </SidebarBox>
         <SidebarBox>
+          <Switch checked={isShowDirection} labelText='Show Direction Mark' leftTheme='off' onChangeCallback={showDirection} rightTheme='on' state='default' />
+          <br />
```

Comment:
It's better to style the switch or to have a wrapper styled for the margin than having a <br/>